src/gcp_scanner/crawl.py

```diff
@@ -528,8 +524,8 @@ def get_bq_tables(project_id: str, dataset_id: str,
     while request is not None:
       response = request.execute()
 
-      for table in response.get("tables", []):
-        list_of_tables.append(table)
+      list_of_tables = [table 
+                        for table in response.get("tables", [])]
```

Comment:
This is redundant. Replace with `list_of_tables = response.get("tables", [])`

Reply:
I have made the requested changes in the latest PR. Check and let me know if any changes required.